plugins/toolbox-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "blockly-toolbox-search",
```

Comment:
This is the wrong name format. The name must be `@blockly/toolbox-search` since this is a first-party plugin. Probably too late now but for future reference you can pass a --first-party flag to the plugin creation script to have it generate the correct names. Make sure you do a search for this string and change it everywhere e.g. in the README. I will send you the link to the rest of the plugin checklist.

Reply:
Thanks for the catch, and good to know about the plugin creation script flag.